scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -1023,11 +1029,91 @@ function scr_initialize_custom() {
+			["display_name" , $"{role[100,10]} {squad_name}"]
+		],
+	    "scout_squad":[
+	        [
+	            role[100,12], 
+	            {"max":9,"min":4, "loadout":{
+	                    "required":{
+	                        "wep1":[wep1[100,12],4], 
+	                        "wep2":[wep2[100,12],4]
+	                    },
+	                    "option" :{
+	                        "wep1":[
+	                            [["Flamer","Plasma Gun",], 1],
+	                            [["Bolter","Stalker Pattern Bolter"], 3],
+	                            [["Missile Launcher","Heavy Bolter"], 1]],
+	                        "wep2":[
+	                            [["Chainsword","Combat Knife"], 5],
+	                        ]
+	                    } 
+	                }
+	            },
+	        ],
+	        [
+	            role[100,18], 
+	            {
+	            	"max":1,"min":1,
+	            	"loadout":{
+		                "option":{
+		                    "wep1":[
+		                        [["Power Sword","Chainsword","Power Axe","Power Fist"],1]
+		                    ],
+		                    "wep2":[
+		                        [["Plasma Pistol","CombiFlamer","Stalker Pattern Bolter","Storm Bolter"],1]
+		                    ]
+		                }
+		            },
+	           	 	"role":$"{role[100,12]} {role[100,18]}",
+	            }   
+	        ],
+	        ["display_name" , $"{role[100,12]} {squad_name}"]
+	    ],
+	    "scout_sniper_squad":[
+	        [
+	            role[100,12], 
+	            {
+	                "max":4,
+	                "min":2, 
+	                "loadout":{
+	                    "required":{
+	                        "wep1":["Sniper Rifle",2], 
+	                        "wep2":["Combat Knife",4]
+	                    },
+	                    "option" :{
+	                        "wep1":[
+	                            [["Sniper Rifle","Stalker Pattern Bolter"], 1],
+	                            [["Missile Launcher","Sniper Rifle","Stalker Pattern Bolter"], 1]
+	                        ],
+	                    }
+	                },
+	                "role":$"{role[100,12]} Sharpshooter", 
+	            },
+	        ], 
+	        [
+	        	role[100,18],
+		        {
+		            "max":1,
+		            "min":1,
+		            "loadout":{
+			            "option":{
+			                "wep1":[
+			                    [["Power Sword","Chainsword","Power Axe","Power Fist"],1]
+			                ],
+			                "wep2":[
+			                    [["CombiFlamer","Stalker Pattern Bolter",],1]
+			                ]
+			            }
+		        	},
+		        	"role":$"{role[100,12]} {role[100,18]}",
+		        }
+	        ],
+	        ["display_name" , $"{role[100,12]} Sniper {squad_name}"]
+	    ]	    			
 	};
-	if (global.chapter_name="Salamanders"){ //salamanders squads
+	if (global.chapter_name=="Salamanders") or (obj_ini.progenitor==8){ //salamanders squads
```

Comment:
we should also have a global list of chapters, maybe as an enum, so that we can stop using hardcoded strings. But this is for another task